systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1388,6 +1401,33 @@ void verifyVolumeNamesAndLabels(int kafkaReplicas, int diskCountPerReplica, int
         }
     }
 
+    void waitForPVCDeletion(int kafkaReplicas, JbodStorage jbodStorage) {
+        TestUtils.waitFor("Wait for PVC deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Constants.TIMEOUT_FOR_RESOURCE_CREATION, () -> {
+            List<String> pvcs = kubeClient().listPersistentVolumeClaims().stream()
+                    .map(pvc -> pvc.getMetadata().getName())
+                    .collect(Collectors.toList());
+            boolean isCorrect = false;
+
+            for (SingleVolumeStorage singleVolumeStorage : jbodStorage.getVolumes()) {
```

Comment:
Wouldn't be better to create list of storages, which shouldn't be deleted after kafka deletion and compare it with `pvcs` list?